src/main/java/org/opensearch/sdk/SDKClient.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.sdk;
+
+import java.io.IOException;
+
+import org.apache.http.HttpHost;
+
+import org.apache.http.conn.ssl.NoopHostnameVerifier;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.opensearch.client.RestClient;
+import org.opensearch.client.RestClientBuilder;
+import org.opensearch.client.json.jackson.JacksonJsonpMapper;
+import org.opensearch.client.opensearch.OpenSearchClient;
+import org.opensearch.client.transport.OpenSearchTransport;
+import org.opensearch.client.transport.rest_client.RestClientTransport;
+
+/**
+ * This class creates a Client for SDK to make requests to OpenSearch
+ */
+public class SDKClient {
+    private final Logger logger = LogManager.getLogger(SDKClient.class);
+    private OpenSearchClient client;
+    private RestClient restClient = null;
+
+    /**
+     * Creates client for SDK
+     * @param hostAddress The address client can connect to
+     * @param port The port of the address
+     * @throws IOException if client failed
+     * @return SDKClient
```

Comment:
What kind of client? You're returning an `OpenSearchClient` not an `SDKClient`. What am I supposed to do with it?  Am I responsible for closing it when I'm done with it?  Call that out here, e.g., "the user is responsible for calling `doClose()` when finished with it." (Except that method is on this class not the `OpenSearchClient` class, so I'm confused.)